.github/workflows/console-ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
```

Comment:
I really like this, creating separate workflow scripts for each integration. Super easy to add new ones!